### PR TITLE
docs: readme update

### PR DESCRIPTION
**Pull request by MarkPieszak**

### CODE REVIEW

**Review by alan-agius4 — DISMISSED** [review]
I am okay with this, although I don't know about adding reference to `ASP.NET core engine` and Hapi engine as these engines are no longer supported.

**Comment by MarkPieszak** [comment]
removed! @alan-agius4 good idea

File: README.md [hunk 1]
```diff
@@ -54,9 +54,13 @@ We believe that using the word "universal" is correct when referring to a JavaSc
 
 - [Adam Plumer](https://github.com/CaerusKaru) and [Fabian Wiles](https://github.com/Toxicable) - Current maintainers
 - [PatrickJS](https://twitter.com/gdi2290) and [Jeff Whelpley](https://twitter.com/jeffwhelpley) - Founders of the Angular Universal project. (Universal rendering is also called [PatrickJS-ing](https://twitter.com/jeffbcross/status/846512930971516928))
-- [Jason Jean](https://github.com/FrozenPandaz) - Express engine and Universal support for CLI
+- [Mark Pieszak](https://twitter.com/MarkPieszak) - Core team
+  - [Angular Universal - Server-side Rendering Deep Dive](https://trilon.io/blog/angular-universal-server-side-rendering-deep-dive)
```
Comment (CaerusKaru) [inline README.md:58]: I'd prefer if we left these out, or maybe move them to an alternate section on "Community Resources"?

Comment (CaerusKaru) [reply README.md:58]: I see there are some other articles here, and if it were in scope for this PR, I'd ask for those to be moved as well. It's not just your articles, we'd move those too if we have the cycles to refactor the README.

Comment (MarkPieszak) [reply README.md]: great idea, fixed this and moved to Community Resources section! @CaerusKaru

**Review by CaerusKaru — APPROVED** [review]

**Comment by MarkPieszak** [comment]
all set! @alan-agius4 :)

**Pull request merged**